src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5044,4 +5053,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_CompilerAndLanguageVersion" xml:space="preserve">
     <value>Compiler version: '{0}'. Language version: {1}.</value>
   </data>
+  <data name="WRN_DefaultInSwitch" xml:space="preserve">
+    <value>Did you mean to use the default switch label (`default:`) rather than `case default:`? If you really mean to use the default literal, consider `case (default):` or another literal (`case 0:` or `case null:`) as appropriate.</value>
+  </data>
+  <data name="WRN_DefaultInSwitch_Title" xml:space="preserve">
+    <value>Did you mean to use the default switch label (`default:`) rather than `case default:`? If you really mean to use the default literal, consider `case (default):` or another literal (`case 0:` or `case null:`) as appropriate.</value>
+  </data>
```

Comment:
I don't know. We have a test that requires warnings to have matching resource strings, and those generally match the original except they don't have substitutions.